### PR TITLE
Console logging for vps

### PR DESCRIPTION
**Pull request by zmc**
New feature works:
http://pulpito.ceph.com/zack-2017-11-08_23:28:17-cephmetrics-master-distro-basic-vps/detail

No regressions:
http://pulpito.ceph.com/zack-2017-11-08_22:56:27-cephmetrics-master-distro-basic-ovh/detail
http://pulpito.ceph.com/zack-2017-11-08_23:58:27-smoke-master-distro-basic-smithi/detail

### CODE REVIEW

**Comment by zmc** [comment]
@dmick do you think you'd have a few minutes to look this over?

File: teuthology/orchestra/console/virtual.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        """
+        Return true if running.
+        """
+        return self.vm_domain.info()[0] == libvirt.VIR_DOMAIN_RUNNING
+
+    def power_cycle(self):
+        """
+        Simiulate virtual machine power cycle
+        """
+        self.vm_domain.info().destroy()
```
Comment (vasukulkarni) [inline teuthology/orchestra/console/virtual.py]: This should change, It shouldn't destroy and recreate for power_cycle, data is gone!

File: teuthology/orchestra/console/virtual.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        """
+        Simiulate hard reset
+        """
+        self.vm_domain.info().destroy()
+
+    def power_on(self):
+        """
+        Simiulate power on
+        """
+        self.vm_domain.info().create()
```
Comment (vasukulkarni) [inline teuthology/orchestra/console/virtual.py]: the power_on/off functions I see are not accurate as compared to physical machines

File: teuthology/orchestra/console/virtual.py (hunk 1 above)
Comment (vasukulkarni) [reply teuthology/orchestra/console/virtual.py]: Sorry this is console only stuff, please ignore

File: teuthology/orchestra/console/virtual.py (hunk 2 above)
Comment (vasukulkarni) [reply teuthology/orchestra/console/virtual.py]: Same here, please ignore!

File: teuthology/orchestra/console/virtual.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+    def power_off(self):
+        """
+        Simiulate power off
```
Comment (badone) [inline teuthology/orchestra/console/virtual.py]: Typo?

**Comment by dmick** [comment]
I guess we can close this, yeah?

**Pull request closed without merging**